### PR TITLE
fix(ui): Release stats chart summary count from api, fixed average

### PR DESCRIPTION
**Pull request by matejminar**
This PR shows total active users/sessions summary from API and fixes session duration averaging bug.

### CODE REVIEW

File: src/sentry/static/sentry/app/views/releasesV2/detail/overview/chart/releaseChart.tsx [hunk 1]
```diff
@@ -39,7 +39,7 @@ class ReleaseChart extends React.Component<Props> {
     const {yAxis} = this.props;
     switch (yAxis) {
       case 'sessionDuration':
-        return typeof value === 'number' ? getExactDuration(value, true) : value;
+        return typeof value === 'number' ? getExactDuration(value, true) : '-';
```
Comment (markstory) [inline src/sentry/static/sentry/app/views/releasesV2/detail/overview/chart/releaseChart.tsx]: Should we use \u2013 here (n-dash) or \u2015 (m-dash) here?

File: src/sentry/static/sentry/app/views/releasesV2/detail/overview/chart/releaseChart.tsx [hunk 2]
```diff
@@ -49,6 +49,26 @@ class ReleaseChart extends React.Component<Props> {
...
+          max: 100,
+          scale: true,
+          axisLabel: {
+            formatter: '{value}%',
+            color: theme.gray1,
+          },
+        };
+      case 'sessionDuration':
+      case 'sessions':
+      case 'users':
```
Comment (markstory) [inline src/sentry/static/sentry/app/views/releasesV2/detail/overview/chart/releaseChart.tsx]: Should these options be an enum?

File: src/sentry/static/sentry/app/views/releasesV2/detail/overview/chart/releaseChart.tsx (hunk 1 above)
Comment (matejminar) [reply src/sentry/static/sentry/app/views/releasesV2/detail/overview/chart/releaseChart.tsx]: Good idea

File: src/sentry/static/sentry/app/views/releasesV2/detail/overview/chart/releaseChart.tsx (hunk 2 above)
Comment (matejminar) [reply src/sentry/static/sentry/app/views/releasesV2/detail/overview/chart/releaseChart.tsx]: Updated 👍

File: src/sentry/static/sentry/app/views/releasesV2/detail/overview/chart/releaseStatsRequest.tsx [hunk 3]
```diff
@@ -282,17 +291,19 @@ class ReleaseStatsRequest extends React.Component<Props, State> {
...
+        responseData
+          .map(([timeframe, values]) => {
+            chartData.data.push({
+              name: timeframe * 1000,
+              value: Math.round(values.duration_p50),
+            });
+
+            return values.duration_p50;
+          })
+          .filter(duration => defined(duration))
```
Comment (markstory) [inline src/sentry/static/sentry/app/views/releasesV2/detail/overview/chart/releaseStatsRequest.tsx]: Is eliding data points going to skew the mean?

**Review by markstory — APPROVED** [review]

File: src/sentry/static/sentry/app/views/releasesV2/detail/overview/chart/releaseStatsRequest.tsx (hunk 3 above)
Comment (matejminar) [reply src/sentry/static/sentry/app/views/releasesV2/detail/overview/chart/releaseStatsRequest.tsx]: Yes, we don’t want datapoints without duration to affect total average.

**Pull request merged**